docs/ossm/ossm2-vs-ossm3.md

```diff
@@ -0,0 +1,103 @@
+
+However, when an Istio resource has a name other than “default” - as required when multiple control plane instances are present and/or a canary-style control plane upgrade is in progress, it is necessary to use a label that indicates which control plane (revision) the workload(s) belong to - namely, `istio.io/rev=<IstioRevision-name>`. These labels may be applied at the workload or namespace level.
+
+## Independently Managed Gateways
+
+In Istio, gateways are used to manage traffic entering (ingress) and exiting (egress) the mesh. While by default, OpenShift Service Mesh 2 deployed and managed an Ingress Gateway and an Egress Gateway with the service mesh control plane, configured in the `ServiceMeshControlPlane` resource, the OpenShift Service Mesh 3 operator will no longer create or manage gateways. 
+
+In OpenShift Service Mesh 3, gateways are created and managed independent of the operator and control plane using gateway injection or Kubernetes Gateway API resources, both of which provide much greater flexibility than was possible with the ServiceMeshControlPlane. This allows the gateways to be customized for a set of applications, deployed and managed with the same lifecycle as their corresponding applications.
+
+This change was made because, as a good practice, gateways are better managed together with their corresponding worklods than with the service mesh control plane. This change also means starting with a gateway configuration that can expand over time to meet the more robust needs of a production environment, which was not possible with the default gateways in OpenShift Service Mesh 2. 
+
+Note that even with this change, gateways may continue to be deployed onto nodes or namespaces independent of applications (for example, a centralized gateway node).  Istio gateways also remain eligible to be deployed on OpenShift infrastructure nodes.
+
+## OpenShift Routes must be explicitely created
+
+An OpenShift `Route` resource allows an application to be exposed with a public URL using OpenShift's Ingress operator for managing HAProxy based Ingress controllers. OpenShift Service Mesh 2 included a feature called Istio OpenShift Routing (IOR) that automatically created and managed OpenShift routes for Istio gateways. While this was convenient, as the operator would manage these routes for the user, it often caused confusion around ownership as many `Route` resources are managed by administrators. The feature also lacked the configurability of an independent `Route` resource, created unnecessary routes, and exhibited unpredictable behavior during upgrades.
+
+Thus, in OpenShift Service Mesh 3, when a `Route` is desired to expose an Istio gateway, it must be created and managed by the user. Note that it is also possible to expose an Istio gateway through a Kubernetes service of type LoadBalancer if a route is not desired.
+
+## Introducing Canary Upgrades
```

Comment:
maybe somewhere in this section we should mention that users will now have to select a patch version (e.g. v1.21.3 instead of just v1.21) and patch updates are not applied automatically anymore. also, we now use upstream Istio versions in the spec.version field instead of OSSM version. customers can see the versions supported by the operator by running
```bash
oc explain istios.spec.version
```

Reply:
I am going to break this topic out into its own issue to add once the automatic upgrade issue is complete and we are sure which version of Istio will ship with GA OSSM 3. It's a little complicated to explain without examples and I want to make sure those are accurate for the GA release.